### PR TITLE
Changes for migration to mongodb from file

### PR DESCRIPTION
**Pull request by sajal-1999**

### CODE REVIEW

File: src/main/java/com/newordle/newordle/dao/WordsDao.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            Set<String> wordSet = new HashSet<>();
+            while (cursor.hasNext()) {
+                wordSet.add((String) cursor.next().get("word"));
+            }
+            return wordSet;
+        } catch (Exception e) {
+            System.out.println("Error fetching words!");
+            return null;
+        } finally {
+            terminateConnection();
```
Comment (arunimaagl) [inline src/main/java/com/newordle/newordle/dao/WordsDao.java:93]: Line never reached. Update your return statement to finally block.

File: src/main/java/com/newordle/newordle/dao/WordsDao.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            Document foundWordDoc = collection.find(Filters.eq("_id", _id)).first();
+            if (foundWordDoc == null) {
+                return null;
+            }
+            return new WordsDb(_id, (String) foundWordDoc.get("word"), (Boolean) foundWordDoc.get("used"));
+        } catch (Exception e) {
+            e.printStackTrace();
+            return null;
+        } finally {
+            terminateConnection();
```
Comment (arunimaagl) [inline src/main/java/com/newordle/newordle/dao/WordsDao.java:74]: Line never reached. Update your return statement to finally block.

**Review by arunimaagl — CHANGES_REQUESTED** [review]
Please make changes mentioned in the comments.

File: src/main/java/com/newordle/newordle/dao/WordsDao.java (hunk 1 above)
Comment (sajal-1999) [reply src/main/java/com/newordle/newordle/dao/WordsDao.java:93]: Ah right. Good catch

**Review by arunimaagl — APPROVED** [review]
Looks good to me!

**Pull request merged**